staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6129,20 +6129,20 @@ type SELinuxOptions struct {
 type WindowsSecurityContextOptions struct {
 	// GMSACredentialSpecName is the name of the GMSA credential spec to use.
 	// +optional
-	GMSACredentialSpecName *string `json:"gmsaCredentialSpecName,omitempty" protobuf:"bytes,1,opt,name=gmsaCredentialSpecName"`
```

Comment:
We still need to have these fields available on the WindowsSecurityContextOptions.

We just want to remove the untilfeature.DefaultFeatureGate.Enabled(kubernetes.WindowsGMSA) checked in kuberuntime_container_windows.go (and anywhere else they are in code)

Reply:
Done

Reply:
Thanks!
I think you need to run hack/update-all.sh (or at least hack/update-codegen.sh) to update the removals in staging/src/k8s.io/api/core/v1/generated.proto and other auto-generated files.